CRC21/ModuleCRC21.sol

```diff
@@ -0,0 +1,34 @@
+pragma solidity ^0.6.8;
+
+import "./ModuleCRC20.sol";
+
+contract ModuleCRC21 is ModuleCRC20  {
+
+    event __CronosSendToChain(address sender, address recipient, uint256 amount, uint256 bridge_fee, uint256 chain_id);
+    event __CronosCancelSendToChain(address sender, uint256 id);
+
+    constructor(string memory denom_, uint8 decimals_) ModuleCRC20(denom_, decimals_) public {
+        decimals = decimals_;
+        denom = denom_;
+    }
+
+    // make unsafe_burn internal
+    function unsafe_burn_internal(address addr, uint amount) internal {
+        // Deduct user's balance without approval
+        require(balanceOf[addr] >= amount, "ds-token-insufficient-balance");
+        balanceOf[addr] = sub(balanceOf[addr], amount);
+        totalSupply = sub(totalSupply, amount);
+        emit Burn(addr, amount);
+    }
```

Comment:
Instead of re-declaring this function, should we update the ModuleCRC20 to make the `unsafe_burn` internal function?

Reply:
okay I am cloned the CRC20 contract and cleaned it up